pyproject.toml

```diff
@@ -1,3 +1,90 @@
+[build-system]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "cpg-flow"
+description = "CPG Flow API for Hail Batch"
+requires-python = ">=3.10"
```

Comment:
Should we change this to 3.12?

Reply:
Are we upgrading to 3.12? The current prod pipes fails on 3.12. It only currently works on 3.10.

I'd be inclined to do the splitting of the code first and then upgrade after.

Reply:
We'll skip the upgrade for 3.12 for now.

Reply:
I've just changed the 3.12 to 3.10 back from that other PR.